src/components/structures/RoomView.js

```diff
@@ -469,7 +470,7 @@ module.exports = React.createClass({
         this.tabComplete.setCompletionList(
             MemberEntry.fromMemberList(members).concat(
                 CommandEntry.fromCommands(SlashCommands.getCommandList())
-            )
+            ).concat(EmojiEntry.entries)
```

Comment:
Hm. Slightly scared about the perf/UX impact of appending eeeeevery emoji ever to the tab complete list all the time - mainly in terms of the CPU required to filter down the list once you start filtering.  I wonder if we should only be adding it in as an option in the special case of the user hitting a colon?  I haven't played with it in anger yet, but definitely worth keeping an eye on CPU...